src/Aop/Support/ModifierMatcherFilter.php

```diff
@@ -47,6 +47,7 @@ class ModifierMatcherFilter implements PointFilter
     public function __construct($initialMask = 0)
     {
         $this->andMask = $initialMask;
+        $this->notMask = \ReflectionMethod::IS_ABSTRACT;
```

Comment:
I feel myself shame for tests :) I'd vote for unit tests for each class and only several functional tests for checking that whole system is working as expected and still able to transform files and perform weaving. 

For this concrete class we can add just simple test case that modifier matcher filter by default doesn't match abstract methods.

One more caution: modifier filter is also used for property matching. Of course, "not abstract" for properties will be always true. But if we will use ModiferFilter for properties with negation, then it can be very unpredictable :)